src/frame/worker/GroupCrypto.ts

```diff
@@ -11,15 +11,15 @@ import {publicKeyToBytes} from "../../lib/Utils";
  * @param {SigningKeyPair}        signingKeys   Current users signing keys used to sign transform key
  * @param {boolean}               addAsMember   Whether to add the current group admin as a member
  */
-export function createGroup(userPublicKey: PublicKey<Uint8Array>, signingKeys: SigningKeyPair, memberList: UserOrGroupPublicKey[]) {
+export function createGroup(signingKeys: SigningKeyPair, memberList: UserOrGroupPublicKey[], adminList: UserOrGroupPublicKey[]) {
```

Comment:
Docs should be updated

Reply:
done

Reply:
Going forward, you can probably just remove all of the @param stuff in this repo. I originally had them there in some situations because I was still coding in my old Javascript ways and hadn't moved my thinking over to Typescript which I was just getting into at that point. Now because all of the argument types are provided in the code with Typescript, I don't think the @param stuff is very useful anymore and, like this comment suggests, makes things very easy to get out of date. 

So going forward, if you modify a function which has the @param stuff in the method comment, feel free to just remove all of it and just leave the method comment only.